jenkins/Jenkinsfile.nightly

```diff
@@ -140,14 +152,15 @@ pipeline {
 						script { echoBanner(STAGE_NAME) }
 						dir('lisk-service') {
 							nvm(readFile('.nvmrc').trim()) {
-								dir('./framework') { sh "yarn run test:functional" }
-								dir('./services/blockchain-app-registry') { sh "yarn run test:functional" }
-								dir('./services/blockchain-connector') { sh "yarn run test:functional" }
-								dir('./services/blockchain-indexer') { sh "yarn run test:functional" }
-								dir('./services/fee-estimator') { sh "yarn run test:functional" }
-								dir('./services/market') { sh "yarn run test:functional" }
-								dir('./services/transaction-statistics') { sh "yarn run test:functional" }
-								dir('./tests') { sh "yarn run test:functional" }
+								sh 'make build-local'
```

Comment:
May be this is unnecessary. We are already doing `make build-local` [here](https://github.com/LiskHQ/lisk-service/blob/76eb5869b44aa62469e92b26f15704ba9f18da1b/jenkins/Jenkinsfile.nightly#L70)